nemo/collections/tts/models/fastpitch.py

```diff
@@ -197,8 +197,6 @@ def parser(self):
     def parse(self, str_input: str, normalize=True) -> torch.tensor:
         if self.training:
             logging.warning("parse() is meant to be called in eval mode.")
-        if str_input[-1] not in [".", "!", "?"]:
-            str_input = str_input + "."
```

Comment:
Is this done by tokenizer now?

Reply:
No. It is not expected behavior for user.

What if I want to run model with text “hello, world;” or “I want to take the following items:”? So, let’s user thinks about it.